cmd/kubepkg/templates/latest/deb/kubelet/lib/systemd/system/kubelet.service

```diff
@@ -1,8 +1,9 @@
 [Unit]
 Description=kubelet: The Kubernetes Node Agent
 Documentation=https://kubernetes.io/docs/home/
-Wants=network-online.target
-After=network-online.target
+Wants=network-online.target time-sync.target
+After=network-online.target time-sync.target
```

Comment:
we need to have a good argument to include time-sync as a dependency for all users out there:
https://www.freedesktop.org/software/systemd/man/systemd.special.html#time-sync.target

would this block / fail if NTP is disabled entirely on a system?

Reply:
AFAIK using ‘Wants=‘ will soft depends if ‘time-sync.service’ is available so will not fail if NTP is not exists; ‘Requires=‘ does the hard enforcement and fail if dependency not meet.